### PR TITLE
added community highlights with Predix

### PR DESCRIPTION
**Pull request by tjsavage**

### CODE REVIEW

**Comment by tjsavage** [comment]
@arthurevans @ebidel @kaycebasques @robdodson WDYT?

**Comment by robdodson** [comment]
LGTM but I already reviewed it once before :D

File: _posts/2016-03-10-community-highlights-predix-ui.markdown [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+---
+
+_In the recurring Community Highlights, we feature the latest neat projects and components being built with Polymer._
+
+[GE’s Predix](https://urldefense.proofpoint.com/v2/url?u=https-3A__www.predix.io_&d=CwMFaQ&c=IV_clAzoPDE253xZdHuilRgztyh_RiV3wUrLrDQYWSI&r=9F6Dmst0NH2RGO9xMmnJKOe7JlNJuxJrlfva7kCPc3E&m=lMhDzKm4KW-DnORnk5NukOAhKd1aSZrcJ0eaHvqXrlc&s=fqkaPf3DIapZyaABDOWbu7JpCkG9SRTnkT07Xrn-vIk&e=><https://urldefense.proofpoint.com/v2/url?u=https-3A__www.predix.io_&d=CwMFaQ&c=IV_clAzoPDE253xZdHuilRgztyh_RiV3wUrLrDQYWSI&r=6IzGNk0NML0ha2ThIeehmg&m=JXx-B86n0-MEQMge5j-Hjt3a7R2RrK72eOmv2fTnFH4&s=s0eE4wQ3tFbQ-z2eIiUuUGGXEBCyH8yf0iooi-4t3RI&e=>) is a cloud-based platform-as-a-service built to support industrial internet of things applications - for analyzing and monitoring the massive stream of data coming from something like a windfarm or smart grid.  GE has been working with Polymer for over a year, and [recently announced Predix UI](https://www.predix.io/blog/article.html?article_id=1705) - a comprehensive set of Polymer-based web components that make it easy to assemble a high-quality applications on top of the Predix platform.
```
Comment (arthurevans) [inline _posts/2016-03-10-community-highlights-predix-ui.markdown]: Looks like the URL is included twice?

1st sentence--seems like the world is standardizing capitalizing this as Internet of Things (IoT). Suggest breaking the sentences after "applications". Maybe start the next sentence, "For example, analyzing ...

2nd sentence, " - " should be an em dash with no space "—"

File: _posts/2016-03-10-community-highlights-predix-ui.markdown [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Predix UI uses Polymer to encapsulate and distribute a cohesively-designed set of elements for everything from charts and data tables to responsive layouts and color schemes. The declarative UI components make it possible to build powerful dashboards with minimal code, while also being useful for complex and highly-customized applications. Predix UI includes a Yeoman-based [component generator](https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_PredixDev_generator-2Dpx-2Dcomp&d=CwMFaQ&c=IV_clAzoPDE253xZdHuilRgztyh_RiV3wUrLrDQYWSI&r=6IzGNk0NML0ha2ThIeehmg&m=tp3Bitzt16t1VIdu8EMWmsqiuR2Tm68BedBg0gw8oDE&s=YqNx5yCVhYpeRViUvrdE2_umD0_O3XGfnyFuasprAbs&e=) as well, to make it as frictionless as possible to create new UI components.
```
Comment (arthurevans) [inline _posts/2016-03-10-community-highlights-predix-ui.markdown]: I might cut the "cohesively-designed" just to tighten up this sentence a bit. Or just go with "cohesive."

2nd sentence with two complicated clauses is a little hard to parse.  Perhaps replace "while also being useful for" with "as well as" so the clauses are parallel: "to build dashboards ... as well as complex applications". Alternately, break the sentence & let the second clause stand alone: "But they can also be used for ..."

File: _posts/2016-03-10-community-highlights-predix-ui.markdown [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+By being based on the web components standards with Polymer, the Predix UI components encapsulate their behavior for ultimate flexibility and modularity - they can be used to build applications from scratch, or to integrate with other web frameworks.
```
Comment (arthurevans) [inline _posts/2016-03-10-community-highlights-predix-ui.markdown]: This line and the next, I am also going to suggest s/ - /—/ because that's just how I roll.

File: _posts/2016-03-10-community-highlights-predix-ui.markdown [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+The Predix UI components are now available for companies to build applications on Predix - check them out at [https://predixdev.github.io/predix-ui/](https://predixdev.github.io/predix-ui/).
```
Comment (arthurevans) [inline _posts/2016-03-10-community-highlights-predix-ui.markdown]: "to build applications on Predix" seems a little redundant. Maybe just say, "Check out the Predix UI components on GitHub"?

Probably not useful to people who aren't using Predix, but that should be pretty clear from the rest of the post...

**Comment by arthurevans** [comment]
Minor editorial feedback. Otherwise LGTM.

**Comment by tjsavage** [comment]
Perfect thank you @arthurevans! Long live the mdash.
